main.go

```diff
@@ -37,6 +38,7 @@ import (
 	// Import all Kubernetes client auth plugins (e.g. Azure, GCP, OIDC, etc.)
 	_ "k8s.io/client-go/plugin/pkg/client/auth"
 
+	render "github.com/openshift-kni/performance-addon-operators/pkg/cmd/render"
```

Comment:
the rename is unneeded, but no big deal. Is this done automatically by the tooling?

Reply:
Good question i have no idea. maybe it's a copy-paste mistake

Reply:
ok, no big deal anyway.